src/main/java/nl/fuchsia/controller/FeitContoller.java

```diff
@@ -0,0 +1,29 @@
+package nl.fuchsia.controller;
+
+import nl.fuchsia.model.Feit;
+import nl.fuchsia.services.FeitService;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import javax.validation.Valid;
+
+@RestController
+@RequestMapping("/feiten")
+public class FeitContoller {
+    private final FeitService feitService;
+
+    public FeitContoller(FeitService feitService) {
+        this.feitService = feitService;
+    }
+
+    /**
+     * Deze methode vangt de geposte json @param feit op. Vervolgens haalt hij een nieuw feitnummer op en set deze in de @param feit.
+     * Dit feit word daarna doorgezet naar de service, zodat deze in de database(list) kan worden opgeslagen.
+     */
```

Comment:
Kijk nog eens goed naar de standaard JavaDoc.
Als je in IntelliJ /** typt, dan genereert die al een goede basis voor je.